provisioning/stream/kubernetes/deploy/api/deployment.yaml

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
myslím si, že tenhle soubor by tu neměl být, mělo by to být v gitignore

Reply:
:heavy_check_mark: